integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkEnvIT.java

```diff
@@ -30,6 +35,8 @@ public abstract class SparkEnvIT extends SparkUtilIT {
   private static final Logger LOG = LoggerFactory.getLogger(SparkEnvIT.class);
   private static final ContainerSuite containerSuite = ContainerSuite.getInstance();
 
+  protected static final String icebergRestServiceName = "iceberg-rest";
+  protected String icebergRestServiceUri = "http://%s:%d/iceberg/";
```

Comment:
it's hacky here to use as a formatter first.

Reply:
updated.